### DISCUSSION TITLE
PHP Server?

### DISCUSSION THREAD

**Discussion by ericvicenti** [rise-tools/rise-tools#100] [Ideas]
We are deciding what language to support next. Should we build a PHP library that will allow you to define UI that will be rendered by the Rise client?

Upvote this issue and we might prioritize PHP next!

Note: PHP has a special place in our hearts as it was the first language of both Rise creators @grabbou and @ericvicenti

**Comment by mfathonyramdhan** [comment]
That's awesome to hear about the special connection with PHP! It sounds like a great idea to prioritize a PHP library next. Looking forward to seeing how it progresses!

**Comment by selase** [comment]
Yes! and make it seamless with Laravel
